src/math_helpers.cpp

```diff
@@ -34,7 +34,10 @@ auto get(const std::string& op_) -> const Operator& {
       // the first argument is discarded (generally a `-1`).
       // A high precedence is used here in order to ensure the unary minus happens before other
       // operations
-      {"m", {.p = 10, .l = false, .f = [](double unused, double x) -> double { return -x; }}}};
+      {"m", {.p = 10, .l = false, .f = [](double unused, double x) -> double {
```

Comment:
I think the alternative to this would be to write the lambda as:

```
[](double /*unused*/, double x) -> double ...
```

That being said, this works and may be the same thing.

Reply:
That looks better. Changed it